### DISCUSSION TITLE
How to monitor RDS performance insights through aws managed grafana?

### DISCUSSION THREAD

**Discussion by tanujit** [grafana/grafana#63198] [Data Sources]
How to monitor the Top SQL's average latency which comes under performance insights of RDS. As I want to monitor the average of avg latency (ms)/call in grafana.

**Comment by VermaPriyanka** [comment]
This [blog](https://aws.amazon.com/blogs/mt/monitoring-amazon-rds-and-amazon-aurora-using-amazon-managed-grafana/) discusses how Amazon Managed Grafana can be used to visualize RDS PI metrics.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This discussion has received an answer, so I am closing it. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
